MPF.Core/Modules/BaseParameters.cs

```diff
@@ -639,7 +639,10 @@ protected bool ProcessBooleanParameter(List<string> parts, string? shortFlagStri
                 i++;
 
                 (string value, long factor) = ExtractFactorFromValue(parts[i]);
-                return (sbyte)(sbyte.Parse(value) * factor);
+                bool isSByte = sbyte.TryParse(value, out sbyte sByteValue);
+                if (!isSByte)
```

Comment:
I hate to do this to you, but merging the `TryParse` line and the `if` line for each of these is something that I'd commonly do. You can also, if you feel like it, you can use the variant that can take the `NumberStyles.AllowHexSpecifier` that may also fix future hex-related issues.